jwt/builder_test.go

```diff
@@ -41,6 +41,10 @@ type testClaims struct {
 	Subject string `json:"sub"`
 }
 
+func (tc1 testClaims) Equal(tc2 testClaims) bool {
```

Comment:
testClaims is comparable, so we don't need to define `.Equal()` or use `EqualFunc` on it